### ISSUE TITLE
Issue opening assure 

### ISSUE THREAD

**Issue by tanishafrancis** [hatem-darweesh/assuremappingtools#52]
I typed sudo ./run-docker but an error message appeared. Here is a piece of what I saw:

"Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-lbY2ttEZPq: Connection refused"

How do I fix this issue?

**Comment by hatem-darweesh** [comment]
I don't know, this never happened to any of the users. 
Might be an issue with your docker installation, or linux drivers issues.

**Comment by Yanlibao0518** [comment]
hello,
@hatem-darweesh 
when i typed sudo ./run-docker but an error message appeared:

”
No specific tag provided using 'master'
No specific workspace path provided. Running without workspace.
access control disabled, clients can connect from any host
./run-docker: line 67: docker: command not found
“
![image](https://user-images.githubusercontent.com/67492300/117576983-754a4600-b11a-11eb-8ef2-65505859be49.png)

can you help with this issue ?

**Comment by hatem-darweesh** [comment]
@Yanlibao0518 , you don't have proper docker installation. 
please make sure that you can pull and run default docker images. 
and then follow the assuremaptools readme instruction for using docker

**Comment by BatuhanUlucay** [comment]
> I typed sudo ./run-docker but an error message appeared. Here is a piece of what I saw:
> 
> "Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-lbY2ttEZPq: Connection refused"
> 
> How do I fix this issue?

Error about /tmp/dbus is fixed when I purged and reinstalled my nvidia driver.
